server.js

```diff
@@ -533,3 +533,17 @@ setInterval(function() {
                 "Warning! All builders are idle, but there are " + queued + " PRs in the queue.");
     });
 }, THIRTY_MINUTES);
+
+setInterval(function() {
+    homu.retrieveSlaves(function(slaves) {
+		for(var slaveName in slaves){
+		    if(slaves[slaveName].hasOwnProperty("runningBuilds")){
```

Comment:
Let's do this instead, and unindent the next lines:
```js
if (!("runningBuilds" in slaves[slaveName])) {
    continue;
}
```

Reply:
Thanks! I'll make these changes, too.